### DISCUSSION TITLE
My game objects won't move...

### DISCUSSION THREAD

**Discussion by GoldenF50** [stride3d/stride#2377] [Q&A] [answered]
https://github.com/stride3d/stride/assets/107926903/26f07706-3049-4679-8abe-6d3c546d3dda

At this point I'm stuck.
I can't figure out why the ball and Pattles won't move, It'll just shake their. I have it in the code to move but it won't. Someone pleas help me.

**Comment by Doprez** [comment, marked as answer]
The problem seems like you are trying to move physics bound objects by the transform component which will be overridden once the physics simulation happens.

after the movements can you try adding `physicsComponent.UpdatePhysicsTransformation();` this will update the physics component based on the current transform data.

**Reply by GoldenF50** [reply]
Thank you, I will try and come back to tell you if it works.

**Reply by GoldenF50** [reply]
Goodness... I thought I should tell you this func threw an error about some bool, but I found out that their is an setting that lets you move the ridged-body though the transform. It's literally right there. The setting "is Kinematic" but the downside is it can Only be moved by the transform so I'm really going to have to fake these physics.(Good thing it's just Pong).

- Thank You, Sincerely
